database/index.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/test')
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', () => { console.log('connected to db!') });
+
+let imageSchema = new mongoose.Schema({
+  fullSizeURL: String,
+  thumbnailURL: String
+})
+
+let productImage = new mongoose.Schema({
+  images: [imageSchema]
+})
+
+let sellerImage = new mongoose.Schema({
+  images: imageSchema
+})
+
+let ProductImage = mongoose.model('ProductImage', productImage)
```

Comment:
ESLint will probably catch this also, where you should use const instead of let since you don't plan on reassigning this variable.